route/grafananet_test.go

```diff
@@ -63,33 +63,25 @@ func TestNewGrafanaNetConfig(t *testing.T) {
 		{otherFile.Name(), true},
 		{schemasFile.Name(), false},
 	}
-	aggregationFileOptions := []option{
-		{"", true},
-		{"some-path-that-definitely-will-not-exist-for-carbon-relay-ng", true},
-		{otherFile.Name(), true},
-		{aggregationFile.Name(), false},
-	}
```

Comment:
Re-added the test cases in https://github.com/grafana/carbon-relay-ng/pull/485/commits/4cad65f3405056163a00399b785ba5c028a64c0c

(The original PR removed all references to the aggregation file in the `NewGrafanaNetConfig` function, so these tests weren't useful anymore. Since the aggregation file argument has been re-added to this function, these test cases were re-added too)